CLI/generate_report.py

```diff
@@ -6,24 +6,25 @@
 from pathlib import Path, PurePath
 
 from CLI.help import global_variables as gv
-from sparkle.solver.ablation import AblationScenario
```

Comment:
No longer required? Why?

Reply:
I'm not sure
It probably either already was removed when I pulled the latest changes or it wasn't needed anymore and I got a flake8 error that made me remove it. But I didn't have a specific reason to remove it

Reply:
Just pulled the latest update and now it is included again